docusaurus/docs/migrating-from-toolkit.md

```diff
@@ -87,8 +89,10 @@ This step will update any npm scripts in the `package.json` file to match the la
 
 ## Next steps
 
-Once the command has finished the migration we recommend looking at the changes introduced and then running each of the npm scripts inside `package.json` to confirm the plugin can be built, tested, signed etc.
+Once the command has finished the migration we recommend looking at the changes introduced and then running `yarn install` followed by each of the npm scripts inside `package.json` to confirm the plugin can be built, tested, signed etc.
```

Comment:
When the migration command finishes running, look at the changes iintroduced and then run 'yarn install'...